src/seedu/addressbook/Main.java

```diff
@@ -80,6 +80,13 @@ private void exit() {
     private void runCommandLoopUntilExitCommand() {
         Command command;
         do {
+        	try{
+        		storage.checkIfFileExist();
```

Comment:
Something looks weird here, directly call the checking function and catch the exception from there? Why don't you just do like this?
```
if (storage.checkIfFileExist()) {
   ui.showFileNotFoundMessage();
   System.exit(0);
}
```
Please think more about handling exceptions, and discuss with your classmates or tutor if necessary.